reedstrm/baking_workflow_and_errors.md

```diff
@@ -0,0 +1,120 @@
+# Baking and Recipes
+
+Baking has become a critical step for publishing - however, it is not planned for it to cause publishing to fail. This is partly due to how long baking may take - much like PDF generation, it can take several hours, if there are external parts, like exercises, that need to be fetched and incorporated into the pages.
```

Comment:
Yes, I think we're saying the same thing, but calling them different things. Content that is published but does not successfully bake will only be available via the special 'baked=False", does not show in search, does not take over 'non-version' urls (latest). So, perhaps "saved but not published" is a better description.